src/static/css/module/prepare-worksheets.less

```diff
@@ -0,0 +1,112 @@
+/* ==========================================================================
+   "Home buyer's preparation worksheets" custom styles
+   ========================================================================== */
+
+/* Preparation worksheet specific styles */
+
+// Width of right-hand worksheet column (priority, likely-ness, etc).
+@right-column-width: 202px;
+
+// Colors
+@btn-grade-disabled-color: @gray-80;
+@btn-grade-unselected-color: @gray-50;
+@btn-grade-selected-color: @darkgray;
+@btn-grade-hover-color: @gray;
+
+@btn-grade-disabled-text-color: @white;
+@btn-grade-unselected-text-color: @darkgray;
+@btn-grade-selected-text-color: @white;
+@btn-grade-hover-text-color: @white;
+
+@btn-delete-color: @gray;
+@btn-delete-hover-color: @gray-20;
+
+// Inputs with buttons for goals, risks, and red flags.
+.input-with-btns_btns { 
+  float: right;
+}
+
+.input-with-btns_input { 
+  display: block;
+  overflow: hidden;
+  padding-right: 10px;
+}
+
+.input-with-btns_input input { 
+  width: 100%;
+}
+
+.input-with-btns {
+  display: block;
+  clear: both;
+  margin-bottom: 10px;
+}
+
+// Preparation worksheets goals, risks, red flag icon colors.
```

Comment:
**Suggestion:** Better selectors for these might be `.worksheets-icon__goals, .worksheets-icon__risk, .worksheets-icon__flag`. They are more declarative and decouples the color from the icon. For example if you wanted to use the non-round versions or a different icon altogether, you can do so without having to change the CSS.